config/user/.eslintrc.base.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": [
+    "./node_modules/kyt/config/.eslintrc.base.json"
```

Comment:
eslint respects local eslintrc files. I tried `root:true` in the root file but that didn't seem to help.